src/components/button/button.tsx

```diff
@@ -26,11 +26,15 @@ export class Button {
   }
 
   componentWillLoad() {
-    if (this.position.toUpperCase() !== "LEFT") {
-      this.position = 'right';
+    if(!this.position) {
```

Comment:
can we have instead of this code in the this.position !== 'right' and on the other side this.position === 'right' ? would save some code (as discussed). lEfT should not be a valid value.

Reply:
The problem with your solution is that if the user types: `position="fgfdgdg"` then this is rendered as invalid, and the icon is not displayed at all. The purpose of making it like this: `this.position.toUpperCase() !== "LEFT"` is to make sure if the user types an invalid value, the position defaults to left. But either way, this is not important. Please consider this resolved, and merge it.